sljit_src/sljitProtExecAllocator.c

```diff
@@ -83,20 +83,16 @@ struct chunk_header {
        as it only uses local variables
 */
 
+#if !(defined(__NetBSD__) && defined(MAP_REMAPDUP))
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdio.h>
```

Comment:
do we need stdio?

Reply:
without this include then P_tmpdir wouldn't be defined.
ironically, most of the time when build for testing; stdio gets included already and it will seem to work, until the code gets deployed in production.